analyzer_test.go

```diff
@@ -55,6 +56,7 @@ func testAnalyzer(t *testing.T, when spec.G, it spec.S) {
 		testCache, err = cache.NewVolumeCache(cacheDir)
 		h.AssertNil(t, err)
 
+		platform := v06.NewPlatform("0.6")
```

Comment:
```suggestion
		platform := platformPkg.NewPlatform("0.6")
```

(not saying these should be the variable names... just that the platform package provides a common entrypoint)